### ISSUE TITLE
[FEATURE REQUEST] Personalized search

### ISSUE THREAD

**Issue by nikolicpetar17** [fow5040/Flow.Launcher.Plugin.SpotifyPremium#21]
When I search for song on spotify it almost always shows up first but when I search for the same song on plugin it shows way below. Is there a way to fix that?

**Comment by nikolicpetar17** [comment]
For example on spotify i only need to type first two letters of the name of song and it shows up first, but in flow launcher when i type in the whole name of the song in only shows in 7th place of results

**Comment by nikolicpetar17** [comment]
Sorry, I just realised that it learns form your previous searches, still it whould be good if it could now your spotify preferences. Thanks for effort put into this plugin btw

**Comment by fow5040** [comment]
I'll take another look at how the search API is structured but I suspect that it doesn't provide the same recommendation order as the client does.

**Comment by nikolicpetar17** [comment]
I've noticed that when I type "Track" before searching it shows me personalized results, but when I just type the name of the song it often won't show up

**Comment by fow5040** [comment]
Yeah the [Spotify search API](https://developer.spotify.com/documentation/web-api/reference/search) still doesn't really provide any insight into how ranking works. We respect the order that is provided for both the ALL search (`sp query`) and the track search (`sp track query`). Maybe the limit that is set (3) in ALL search affects how results are ranked, I could try bumping this up to 10 or so and seeing if results returned line up more with the regular track API.
